main.py

```diff
@@ -1,10 +1,40 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as json_file:
+        data = json.load(json_file)
+    for nickname, user in data.items():
+        race_info = user["race"]
+        race, created = Race.objects.get_or_create(
+            name=race_info["name"],
+            description=race_info["description"]
+        )
+        for skill_ in race_info["skills"]:
+            Skill.objects.get_or_create(
+                name=skill_["name"],
+                bonus=skill_["bonus"],
+                race=race
+            )
+        guild = None
+        if user["guild"]:
+            name = user["guild"]["name"]
+            description = user["guild"]["description"]
```

Comment:
There is no need to allocate these variables, pass values directly to the `get_or_create` method.